tests/baselines/reference/tsserver/plugins/when-plugins-use-LS-to-get-program-and-update-is-pending.js

```diff
@@ -0,0 +1,318 @@
+currentDirectory:: / useCaseSensitiveFileNames: false
+Info seq  [hh:mm:ss:mss] Provided types map file "/typesMap.json" doesn't exist

```

Comment:
Is this a problem?

Reply:
No.. Its used only in some of the TI test cases and there that file is on the host